### PR TITLE
V3 publishing

### PR DESCRIPTION
**Pull request by epananth**
Summary of the changes (Less than 80 chars)
 - Use v3 publishing in aspnetcore 

test build =>  https://dnceng.visualstudio.com/internal/_build/results?buildId=808539&view=results
without default channel => https://dev.azure.com/dnceng/internal/_build/results?buildId=809506&view=logs&s=380ab8c6-4139-55ff-15c4-fb48260f4dca

Addresses #bugnumber (in this specific format)

### CODE REVIEW

**Comment by epananth** [comment]
should be merged after -> https://github.com/dotnet/aspnetcore/pull/25777

**Comment by epananth** [comment]
when RC2 will be merged to master. Master will have V3 changes

**Review by mmitche — APPROVED** [review]

**Comment by mmitche** [comment]
@dougbu Any concerns?

**Review by jcagme — APPROVED** [review]

File: eng/Publishing.props [hunk 1]
```diff
@@ -1,5 +1,9 @@
 <Project>
   <PropertyGroup>
+    <PropertyGroup>
```
Comment (dougbu) [inline eng/Publishing.props:3]: Typo that will break official builds as soon as you check this in

**Review by dougbu — CHANGES_REQUESTED** [review]

File: eng/Publishing.props (hunk 1 above)
Comment (mmitche) [reply eng/Publishing.props:3]: Ya. Looks like the official test build didn't have this typo.

Comment (epananth) [reply eng/Publishing.props]: Good catch! I fixed that.

File: eng/Publishing.props [hunk 2]
```diff
@@ -1,5 +1,8 @@
 <Project>
   <PropertyGroup>
+    <PublishingVersion>3</PublishingVersion>
+  </PropertyGroup>
```
Comment (dougbu) [inline eng/Publishing.props:4]: No, we don't want the end tag here. That'll break the settings below.

**Review by dougbu — CHANGES_REQUESTED** [review]

**Review by dougbu — APPROVED** [review]

**Comment by dougbu** [comment]
@Pilchie approved for RC2❔

From servicing exercise, it appears this is required when publishing w/ latest Arcade bits though I'm not sure why. @mmitche and @epananth why❔

**Review by Pilchie — APPROVED** [review]

**Comment by Pilchie** [comment]
Approved for RC2, yes.

**Pull request merged**